tests/slow/src/test/scala/tests/sbt/SbtBloopLspSuite.scala

```diff
@@ -453,7 +453,7 @@ class SbtBloopLspSuite
       _ <- assertDefinitionAtLocation(
         "build.sbt",
         "sc@@alaVersion := \"2.12.11\"",
-        ".metals/readonly/sbt/Keys.scala",
+        "sbt/Keys.scala",
```

Comment:
Shouldn't this still be in `.metals/readonly`?

Reply:
They still are in `.metals/readonly` but the path became much longer - `.metals/readonly/dependencies/sbt-${version}-sources.jar/sbt/Keys.scala`.
Honestly, it was easier to fix tests by leaving these paths without full prefix than investigating the right one per every test. Anyway, there isn't a lot of meaning in seeing them.

Also, I think it might be annoying to update jar names in these paths in case if some dependency version will be changed in tests

Reply:
Yea good point about changing these in the future. 👍🏼